src/Collection.php

```diff
@@ -17,601 +17,250 @@ class Collection
+     * @throws InvalidArgumentException
      *
-     * @param array $options Optional parameters
-     * @return CollectionMapping
      */
-    public function getMapping(array $options = [])
+    public function exists($index, $collection, array $options = [])
     {
-        return $this->collectionMapping()->refresh($options);
-    }
+        if (empty($index) || empty($collection)) {
+            throw new InvalidArgumentException('Kuzzle\Collection::exists: index and collection name required');
+        }
 
-    /**
-     * Retrieves the current specifications of this collection
-     *
-     * @param array $options Optional parameters
-     * @return mixed
-     */
-    public function getSpecifications(array $options = [])
-    {
-        $data = [
-            'index' => $this->index,
-            'collection' => $this->collection
+        $options['httpParams'] = [
+            ':index' => $index,
+            ':collection' => $collection
         ];
 
         $response = $this->kuzzle->query(
-            $this->buildQueryArgs('collection', 'getSpecifications'),
-            $this->kuzzle->addHeaders($data, $this->headers),
+            $this->kuzzle->buildQueryArgs('collection', 'exists'),
+            [
+                'index' => $index,
+                'collection' => $collection
+            ],
             $options
         );
 
         return $response['result'];
     }
 
     /**
-     * Create the provided documents
+     * Returns the complete list of realtime and stored data collections in
+     * requested index sorted by name in alphanumerical order.
      *
-     * @param array $documents Array of documents to create
+     * @param string $index Index name
      * @param array $options Optional parameters
-     * @return mixed
-     */
-    public function mCreateDocument($documents, array $options = [])
-    {
-        if (!is_array($documents)) {
-            throw new InvalidArgumentException('Collection.mCreateDocument: documents parameter format is invalid (should be an array of documents)');
-        }
-
-        $documents = array_map(function ($document) {
-            return $document instanceof Document ? $document->serialize() : $document;
-        }, $documents);
-
-        $data = ['body' => ['documents' => $documents]];
-
-        $response = $this->kuzzle->query(
-            $this->buildQueryArgs('document', 'mCreate'),
-            $this->kuzzle->addHeaders($data, $this->headers),
-            $options
-        );
-
-        return $response['result'];
-    }
-
-    /**
-     * Create or replace the provided documents
      *
-     * @param array $documents Array of documents to create or replace
-     * @param array $options Optional parameters
      * @return mixed
+     *
+     * @throws InvalidArgumentException
      */
-    public function mCreateOrReplaceDocument($documents, array $options = [])
+    public function listCollections($index, array $options = [])
```

Comment:
why not `list` like in the api and in go?

Reply:
Because list is a reserved name in PHP 5.X so Travis Build will break.